scripts/jenkins-build.bat

```diff
@@ -0,0 +1,65 @@
+@echo off
+REM Build with Jenkins CI.  This script is intended for use with the OME
+REM Jenkins CI infrastructure, though it may be useful as an example for
+REM how one might use various cmake options.
+
+REM So that statements like "set" work inside nested conditionals
+setlocal ENABLEDELAYEDEXPANSION
+setlocal ENABLEEXTENSIONS
+
+set "script=%0"
+
+set "workspace=%~dp0\.."
```

Comment:
I'll add to this PR with another commit.  So long as the job runs `source\scripts\jenkins-build` in a windows cmd shell that should be sufficient.